windows.yml

```diff
@@ -10,5 +10,6 @@ steps:
 
   - script: |
      cd test
+     call test\msvctest-build.bat
```

Comment:
This should be nmake /F MAKEFILENAME Its silently failing  in [CI  tests](https://dev.azure.com/moonjit/moonjit/_build/results?buildId=235&view=logs&j=805631b4-8102-53fd-e75f-25423b543c0c&t=a0980f0c-d3df-5f8b-3308-b96e53d3b0e1&l=12) as path no found.

Reply:
done.